scripts/clean-local-repos.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+repos_dir="${HOME}/repos"
```

Comment:
```suggestion
repos_dir="${1:-.}"
```

I don't think assuming `~/home` is nice, because it would make it only compatible with my dotfiles. Not all people use `~/home`.

How about you use the current directory instead? It would give users more flexibility.